.github/workflows/docker_test.yml

```diff
@@ -27,15 +43,23 @@ jobs:
     name: Test docker images
     steps:
       - uses: actions/checkout@v4
+      - name: "Setup Environment"
+        uses: ./.github/actions/setup-uv
+      - name: Install the project
+        run: uv sync --dev
+      - name: Debug Inputs
+        run: |
+          echo "Image Name: ${{ inputs.image_name }}"
+          echo "Tag Name: ${{ inputs.tag_name }}"
       - name: Build image
         run: |
-          docker build -t langflowai/langflow:latest-dev \
+          docker build -t ${{ inputs.image_name }}:${{inputs.tag_name}} \
             -f docker/build_and_push.Dockerfile \
             .
       - name: Test image
         run: |
           expected_version=$(cat pyproject.toml | grep version | head -n 1 | cut -d '"' -f 2)
```

Comment:
I would actually use `uv` for this, for consistency. See how it's done here: https://github.com/langflow-ai/langflow/blob/main/.github/workflows/release.yml#L132

(Take note on whether you want to remove the leading `v` or not, and which awk arg to print, depending on whether it's `base` or not)

Reply:
Also note that if you're running the `base` image, the process to get the version is different, so you may need to add a `build_type` input, which you can then do a conditional on that says
```
if <build_type == base>: # pseudocode
          version=$(uv tree | grep 'langflow-base' | awk '{print $3}' | sed 's/^v//')
elif <build_type == main> # pseudocode
                  version=$(uv tree | grep 'langflow' | grep -v 'langflow-base' | awk '{print $2}' | sed 's/^v//')

Reply:
for docker testeing we can remove the v in both base and main right?

Reply:
Yeah, pretty sure the docker images don't include the `v` (in both base and main).